frontend/components/MainPage/sections/RewardsSection/StakingRewardsThisEpoch.tsx

```diff
@@ -55,22 +56,34 @@ const useEpochEndTime = () => {
 
 export const StakingRewardsThisEpoch = () => {
   const { data: epochEndTimeInMs } = useEpochEndTime();
-  const { activeStakingProgramMeta } = useStakingProgram();
+  const {
+    activeStakingProgramMeta,
+    activeStakingProgramId,
+    isActiveStakingProgramLoaded,
+  } = useStakingProgram();
+
+  const stakingProgramMeta = activeStakingProgramMeta;
 
   return (
     <Text type="secondary">
       Staking rewards this epoch&nbsp;
       <Popover
         arrow={false}
         content={
-          <div style={{ maxWidth: POPOVER_WIDTH_MEDIUM }}>
-            The epoch for {activeStakingProgramMeta?.name} ends each day at ~{' '}
-            <Text className="text-sm" strong>
-              {epochEndTimeInMs
-                ? `${formatToTime(epochEndTimeInMs * 1000)} (UTC)`
-                : '--'}
-            </Text>
-          </div>
+          isActiveStakingProgramLoaded && activeStakingProgramId ? (
```

Comment:
small fix as this was broken after account creation with no active staking contract